src/Exceptions/HttpGone.php

```diff
@@ -8,4 +8,3 @@ class HttpGone extends HttpException {
     public const DESCRIPTION = "Indicates that access to the target resource is no longer available at the origin server and that this condition is likely to be permanent.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/410";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces